channels/applicationinsights-channel-js/src/Sender.ts

```diff
@@ -724,11 +754,19 @@ export class Sender extends BaseTelemetryPlugin implements IChannelControlsAI {
              * Resend payload. Adds payload back to the send buffer and setup a send timer (with exponential backoff).
              * @param payload
              */
-            function _resendPayload(payload: string[], linearFactor: number = 1) {
+            function _resendPayload(payload: string[], linearFactor: number = 1, endpointUrl: string, status?: number) {
                 if (!payload || payload.length === 0) {
                     return;
                 }
         
+                var endpointHost = urlParseUrl(endpointUrl).hostname;
+                if (_self._statsbeat && _self._statsbeat.isInitialized()) {
+                    _self._statsbeat.countRetry(endpointHost);
```

Comment:
I think `statsbeat` metrics are mutually exclusive. Meaning if status code is `429`, it will count as throttle instead of retry.